package.json

```diff
@@ -63,5 +65,16 @@
     "webpack": "^4.29.6",
     "webpack-cli": "^3.3.0",
     "webpack-dev-server": "^3.2.1"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run test:ci && lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{ts,tsx}": [
```

Comment:
Why no .js extension? kind of pointless since we haven't even moved to typescript yet no?

Reply:
Good catch! My bad, I'll fix that.

Reply:
From huksy docs for glob pattern I chose
> If the glob pattern contains no slashes (/), micromatch's matchBase option will enabled, so globs match a file's basename regardless of directory:
"*.js" will match all JS files, like /test.js and /foo/bar/test.js